src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -542,7 +552,7 @@ void EnBom_Update(Actor* thisx, PlayState* play) {
                 thisx->params = ENBOM_1;
                 this->timer = 10;
                 thisx->flags |= (ACTOR_FLAG_20 | ACTOR_FLAG_100000);
-                this->actionFunc = func_808715B8;
+                this->actionFunc = EnBom_Explode;
```

Comment:
Considering it explodes right after having its params set to ENBOM_1, and it checks for that before setting the AT attack collider, I suspect ENBOM_1 means its exploding, can probably be named.

Reply:
Oh yeah in OoT ENBOM_0 is called BOMB_BODY, and ENBOM_1 is called BOMB_EXPLOSION.

Reply:
Added in latest commit.